en/backups.md

```diff
@@ -0,0 +1,660 @@
+# Backups
+
+There are many ways to backup an Ubuntu installation. The most important thing
+about backups is to develop a *backup plan* consisting of what to backup,
+where to back it up to, and how to restore it.
+
+The following sections discuss various ways of accomplishing these tasks.
+
+# Shell Scripts {#backup-shellscripts}
+
```

Comment:
May need to check heading styles. This page has multiple H1s. Perhaps that is a carry-over from the original?